jaq-json/src/lib.rs

```diff
@@ -512,12 +517,12 @@ impl core::ops::Add for Val {
                 Ok(Str(buf.into(), tag))
             }
             (Arr(mut l), Arr(r)) => {
-                //std::dbg!(Rc::strong_count(&l));
-                Rc::make_mut(&mut l).extend(r.iter().cloned());
+                //std::dbg!(Ptr::strong_count(&l));
```

Comment:
Shouldn't this be `RefPtr` here as well?

Reply:
Yea, good catch. Probably died in some search&replace and did not show up as it is uncommented.